src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_ISymbolInitializer.vb

```diff
@@ -458,5 +458,1362 @@ IFieldInitializerOperation (Field: C.s1 As System.Byte) (OperationKind.FieldInit
+        Return 0
+    End Function
+End Class]]>.Value
+
+            Dim expectedFlowGraph = <![CDATA[
+Block[B0] - Entry
+    Statements (0)
+    Next (Regular) Block[B1]
+Block[B1] - Block
+    Predecessors: [B0]
+    Statements (1)
+        ISimpleAssignmentOperation (OperationKind.SimpleAssignment, Type: null, IsImplicit) (Syntax: '= M()')
+          Left: 
+            IPropertyReferenceOperation: Property C.P1 As System.Int32 (Static) (OperationKind.PropertyReference, Type: null, IsImplicit) (Syntax: '= M()')
+              Instance Receiver: 
+                null
+          Right: 
+            IInvocationOperation (Function C.M() As System.Int32) (OperationKind.Invocation, Type: System.Int32) (Syntax: 'M()')
+              Instance Receiver: 
+                null
+              Arguments(0)
+
+    Next (Regular) Block[B2]
+Block[B2] - Exit
+    Predecessors: [B1]
+    Statements (0)
+]]>.Value
+
+            Dim expectedDiagnostics = String.Empty
+
+            VerifyFlowGraphAndDiagnosticsForTest(Of EqualsValueSyntax)(source, expectedFlowGraph, expectedDiagnostics)
+        End Sub
+
+        <CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)>
+        <Fact>
+        Public Sub NoControlFlow_NonConstantPropertyInitializer_InstanceProperty()
+            Dim source = <![CDATA[
+Class C
+    Public Property P1 As Integer = M()'BIND:"= M()"
+    Public Shared Function M() As Integer
+        Return 0
+    End Function
+End Class]]>.Value
+
+            Dim expectedFlowGraph = <![CDATA[
+Block[B0] - Entry
+    Statements (0)
+    Next (Regular) Block[B1]
+Block[B1] - Block
+    Predecessors: [B0]
+    Statements (1)
+        ISimpleAssignmentOperation (OperationKind.SimpleAssignment, Type: null, IsImplicit) (Syntax: '= M()')
+          Left: 
+            IPropertyReferenceOperation: Property C.P1 As System.Int32 (OperationKind.PropertyReference, Type: null, IsImplicit) (Syntax: '= M()')
+              Instance Receiver: 
+                IInstanceReferenceOperation (ReferenceKind: ContainingTypeInstance) (OperationKind.InstanceReference, Type: C, IsImplicit) (Syntax: '= M()')
+          Right: 
+            IInvocationOperation (Function C.M() As System.Int32) (OperationKind.Invocation, Type: System.Int32) (Syntax: 'M()')
+              Instance Receiver: 
+                null
+              Arguments(0)
+
+    Next (Regular) Block[B2]
+Block[B2] - Exit
+    Predecessors: [B1]
+    Statements (0)
+]]>.Value
+
+            Dim expectedDiagnostics = String.Empty
+
+            VerifyFlowGraphAndDiagnosticsForTest(Of EqualsValueSyntax)(source, expectedFlowGraph, expectedDiagnostics)
+        End Sub
+
+        <CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)>
+        <Fact>
+        Public Sub NoControlFlow_AsNewPropertyInitializer()
+            ' Note: As New cannot be applied to multiple properties, and leads to parse error with no AsNewClauseSyntax node in the tree.
```

Comment:
>' Note: As New cannot be applied to multiple properties, and leads to parse error with no AsNewClauseSyntax node in the tree. [](start = 12, length = 125)

I think it might be possible to get to this scenario with "WithEvents" declaration

Reply:
See MultipleWithEventsInitializers test in this file

---
In reply to: [187468495](https://github.com/dotnet/roslyn/pull/26763#discussion_r187468495) [](ancestors = 187468495)

Reply:
Added `NoControlFlow_WithEventsAsNewPropertyInitializer`

---
In reply to: [187469095](https://github.com/dotnet/roslyn/pull/26763#discussion_r187469095) [](ancestors = 187469095,187468495)